src/go/configgenerator/route_generator_test.go

```diff
@@ -3487,7 +3487,7 @@ func getOverSizeRegexForTest() string {
 	overSizeRegex := ""
 	for i := 0; i < 333; i += 1 {
 		// Use "/**" as it is a replacement token for wildcard uri template.
-		overSizeRegex += "/**"
+		overSizeRegex += "/**/a"
```

Comment:
Why we add an `a` in the end? Could you give explanation? Also it doesn't match the comment?

Reply:
Fixed comment. Changed regex.

Previous regex could be simplified (coalesce `*` characters). New regex cannot be simplified easily, so it will execute the correct branch.